### DISCUSSION TITLE
opinions for specific judge

### DISCUSSION THREAD

**Discussion by raminass** [freelawproject/courtlistener#3369] [Q&A]
Hey I'm trying to pull all supreme court for specific judge, however it return's all opinions by all judges. 
My request as follows:
https://www.courtlistener.com/api/rest/v3/opinions/?cluster__docket__court__id=scotus&author_id=1713

**Comment by mlissner** [comment]
I think you're close, but not quite there. I think the query you want is:

https://www.courtlistener.com/api/rest/v3/opinions/?cluster__docket__court=scotus&author=1713

That just pulls off the `_id` fields from both. I think the ID field was probably fine on the court field, but removing it should help with your performance (and our sad, horribly mistreated servers).
